sled-agent/src/services.rs

```diff
@@ -1233,6 +1345,8 @@ impl ServiceManager {
             addresses,
             gz_addresses: vec![],
             services,
+            enable_ipv6_forwarding: true,
+            enable_ipv6_routing: true,
```

Comment:
Is this a configuration option we *want* to set for all zones? Or is it something that the switch zone should just know to do to itself?

Reply:
> should just be a part of the switch zone starting up

I'm having trouble parsing this - do you mean something that the switch zone services do themselves, or something the sled-agent does using something like `run_cmd`?

Reply:
Summarizing some offline discussion:

- Presumably, we always want these options set for the switch zone
- Therefore, it's static data we can bake into the switch zone image
- and so it doesn't *need* to be a configuration option

(but if it is to unblock dogfood, that's honestly fine)

Reply:
Done, see eb3e63427cb41b5a4331dea8569a7fc09d57640a